common/libs/nodes/rpc/index.ts

```diff
@@ -63,39 +62,29 @@ export default class RpcNode implements INode {
   ): Promise<TokenValue[]> {
     return this.client
       .batch(tokens.map(t => this.requests.getTokenBalance(address, t)))
-      .then(response => {
-        return response.map(item => {
-          // FIXME wrap in maybe-like
-          if (item.error) {
-            return TokenValue('0');
-          }
-          return TokenValue(item.result);
-        });
-      });
-    // TODO - Error handling
+      .then(response => response.map(item => TokenValue(item.result)));
```

Comment:
Error handling TODO is removed, but its not clear to me that there is in-fact error handling here. What happens when only one token request fails, but all others succeeds?

Reply:
My thinking here is that rather than softly resolving the error by hardcoding a TokenValue('0'), we would throw and error so we don't confuse the user. But I guess if a single call fails, then the rest of the calls would subsequently stop as well. What I can do is add an optional overloaded callback that serves as a custom resolver for isValidEthCall.